saleor/graphql/api.py

```diff
@@ -269,14 +269,15 @@ def resolve_address_validator(self, info, input):
 
 
 class Mutations(graphene.ObjectType):
-    assign_navigation = AssignNavigation.Field()
+    address_create = AddressCreate.Field()
+    address_update = AddressUpdate.Field()
+    address_delete = AddressDelete.Field()
 
-    token_create = CreateToken.Field()
-    token_refresh = graphql_jwt.Refresh.Field()
-    token_verify = VerifyToken.Field()
+    assign_navigation = AssignNavigation.Field()
```

Comment:
This should be renamed to `navigation_assign`, according to the <enitity><action> rule.